Source/Urho3D/Core/Attribute.h

```diff
@@ -49,8 +49,10 @@ enum AttributeMode
     AM_COMPONENTID = 0x20,
     /// Attribute is a node ID vector where first element is the amount of nodes.
     AM_NODEIDVECTOR = 0x40,
+    /// Attribute is readonly.
+    AM_READONLY = 0x80,
     /// Attribute is readonly. Can't be used with binary serialized objects.
-    AM_FILEREADONLY = 0x81,
```

Comment:
I prefer leaving extra `,` at the end of last enum item as next time adding new enum element requires single line mod instead of two. Just an observation :)